config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
```

Comment:
- [ ] It would be nice if you add the following commands in the **Install** section to make your readme useful and more descriptive :blush:

## Install
```
bundle install
rails db:create
rails db:migrate
```